### PR TITLE
Adding examples for store fields in FAQs

### PR DESCRIPTION
**Pull request by sreekanth-cb**

### CODE REVIEW

**Comment by sreekanth-cb** [comment]
Tagging @amarantha-k , @abhinavdangeti  for reviews..

**Comment by amarantha-k** [comment]
@sreekanth-cb @abhinavdangeti - Submitted editorial changes, please take a look and confirm if the content is good to merge.

File: modules/fts/pages/fts-troubleshooting.adoc [hunk 1]
```diff
@@ -232,7 +232,7 @@ For example, if you want to retrieve the field "description" in the document, th
 [#fts-type-mappings-child-field]
 image::fts-type-mappings-child-field-dialog-complete.png[,460,align=left]
 
-. *Searching* - you need to specify the fields to be retrieved as a part of the "Fields" option in the search request. This field can take an array of field names which must be loaded and returned with the search response. The field names are specified as string values. While there is no field name pattern matching available, you can use an asterisk "*" to specify that all the fields must be loaded and returned with the response.
+. *Searching* - you need to specify the fields to be retrieved as a part of the "Fields" option in the search request. This field can take an array of field names which must be loaded and returned with the search response. The field names are specified as string values. While there is no field name pattern matching available, you can use an asterisk (*) to specify that all the fields must be loaded and returned with the response.
```
Comment (abhinavdangeti) [inline modules/fts/pages/fts-troubleshooting.adoc]: Would recommend editing this to  ..
```
+ . *Searching* - you need to specify the fields to be retrieved in the "fields" setting within the search request. This setting takes an array of field names which will be returned as part of the search response. The field names are to be strings. While there is no field name pattern matching available, you can use an asterisk ("*") to specify that all stored fields be returned with the response.
```

Comment (amarantha-k) [reply modules/fts/pages/fts-troubleshooting.adoc]: Thanks. Updated.

**Review by amarantha-k — APPROVED** [review]

**Pull request merged**